app/models/fax_recipient.rb

```diff
@@ -1,12 +1,17 @@
 class FaxRecipient
   attr_accessor :residential_address
 
-  def initialize(residential_address:)
+  def initialize(residential_address:, office_location: "")
     @residential_address = residential_address
+    @office_location = office_location
   end
 
   def number
-    self.class.find_by(zip: residential_address.zip)["fax_number"]
```

Comment:
__not blocking__ - I feel like we're *possibly* reaching the inflection point where making this take the snap application itself isn't unreasonable; as well as composing it into the snap application as `snap_appliction.fax_recipient`

Reply:
good idea! updated at 831cbb5